packages/flutter/lib/src/painting/notched_shapes.dart

```diff
@@ -53,9 +53,12 @@ class CircularNotchedRectangle extends NotchedShape {
   ///
   /// The notch is curve that smoothly connects the host's top edge and
   /// the guest circle.
+  ///
+  /// Parameter [inverted] allows for the inversion of the circular notch
+  /// so that it may be applied to the bottom of a path when set to true.
```

Comment:
small wording suggestion here:

```suggestion
  /// If [inverted] is true, the notch is placed at the bottom of the
  /// rectangle.
```